app/controllers/development_controller.rb

```diff
@@ -274,6 +281,15 @@ def test_discussion
     @test_discussion
   end
 
+  def test_discussions
+    100.times do
+      Discussion.create!(title: Faker::Company.bs,
```

Comment:
Happy to consider improvements to development controller in another branch.
I said to @Hsalmon that I'm happy for it to get shitty then get cleaned up
later

On Fri, 7 Aug 2015 at 14:54 James Kiesel <notifications@github.com> wrote:

> In app/controllers/development_controller.rb
> <https://github.com/loomio/loomio/pull/2380#discussion_r36487482>:
>
> > @@ -274,6 +281,15 @@ def test_discussion
> >      @test_discussion
> >    end
> >
> > +  def test_discussions
> > +    100.times do
> > +      Discussion.create!(title: Faker::Company.bs,
>
> As a side note, I've noticed we're basically duplicating FactoryGirl stuff
> in the development controller, do you think there might be a way to include
> those methods here, so this could just be
>
> 100.times { create(:discussion, group: group }
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2380/files#r36487482>.
>